src/datepicker/themes/bs/bs-daterangepicker-container.component.ts

```diff
@@ -49,6 +50,8 @@ export class BsDaterangepickerContainerComponent extends BsDatepickerAbstractCom
     super();
     this._effects = _effects;
 
+    this._customRangesFish = this._config.ranges;
```

Comment:
why fish? plus this `_customRangesFish` field is public, so it should be without underscore

Reply:
Done.